roads.mss

```diff
@@ -3100,3 +3100,87 @@ tertiary is rendered from z10 and is not included in osm_planet_roads. */
     }
   }
 }
+#railways-text-name {
+  /* Render railway=rail from z11 */
+  [railway = 'rail'][zoom >= 11] {
+    text-name: "[name]";
+    text-fill: #333333;
+    text-size: 8;
+    text-dy: 4;
+    text-spacing: 300;
+    text-clip: false;
+    text-placement: line;
+    text-face-name: @book-fonts;
+  }
+  /* Others started from z13. */
+  [railway = 'rail'],
+  [railway = 'subway'],
+  [railway = 'narrow_gauge'],
+  [railway = 'light_rail'],
+  [railway = 'preserved'],
+  [railway = 'funicular'],
+  [railway = 'monorail'],
+  [railway = 'turntable'],
+  [railway = 'tram'] {
+    [zoom >= 13] {
+      text-name: "[name]";
+      text-fill: #333333;
+      text-size: 8;
+      text-dy: 4;
+      text-spacing: 300;
+      text-clip: false;
+      text-placement: line;
+      text-face-name: @book-fonts;
+      text-halo-radius: @standard-halo-radius;
+      text-halo-fill: #cecece;
+    }
+    [zoom >= 14] {
+      text-size: 9;
```

Comment:
The same text size problem as with https://github.com/gravitystorm/openstreetmap-carto/pull/2687/files#r129007651.